deployment-manager/yaml/deployment-manager-config.yaml

```diff
@@ -0,0 +1,69 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+resources:
+- name: umig-test-vm-1
```

Comment:
Should you include some CHANGE-THIS-TO-BLAH lines here?

Reply:
Yes added some comments.